### PR TITLE
Added a Change Log

### PR DESCRIPTION
**Pull request by liangfok**
resolves https://github.com/RobotLocomotion/drake/issues/1882

Obviously, all historical and future PRs should be added to CHANGELING.md but this is a start.

### CODE REVIEW

**Comment by liangfok** [comment]
I just tried to assign @wxmerkt to perform the first round review of this PR but his name was not showing up in the "Assignee" list. Not sure why.

**Comment by RussTedrake** [comment]
He’s not a member of any of the RobotLocomotion teams.  Perhaps we should have a team for the open-humanoids collaborators?

**Comment by wxmerkt** [comment]
Looks great, thanks for adding! Should we use links to the PR/issues for ease of use? You can e.g. use https://github.com/pcreux/pimpmychangelog for that or insert ``[#1953](link)`` manually.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Change Log
+====================
+
+Introduction
+------------
+Drake is undergoing continuous development and some of the changes may break its API and ABI. To help downstream users cope, this document provides brief summaries of the changes. The changes are organized by pull requests that are merged into Drake's master branch.
```
Comment (jwnimmer-tri) [inline CHANGELOG.md]: We should not even pretend to be ABI stable.  It is not within our reach at the moment.

Comment (david-german-tri) [inline CHANGELOG.md]: Agree with @jwnimmer-tri.  Proposed alternative wording: "Drake is undergoing continuous development and cannot yet offer a stable, documented API or ABI."

Comment (liangfok) [inline CHANGELOG.md]: Done. Replaced with @david-german-tri's suggested text.

Comment (liangfok) [inline CHANGELOG.md]: @wxmerkt: Thanks for pointing out that [pimpmychangelog](https://github.com/pcreux/pimpmychangelog) tool. I've modified this PR include its modifications.

**Comment by liangfok** [comment]
@wxmerkt: Thanks for pointing out that [pimpmychangelog](https://github.com/pcreux/pimpmychangelog) tool. I've modified this PR include its modifications.

**Comment by wxmerkt** [comment]
Great, thanks! You can also add another (empty) commit to auto-close the issue once this PR merges:
```
git commit --allow-empty -m "Resolves RobotLocomotion#1882"
```

**Comment by liangfok** [comment]
@wxmerkt: I put a "Resolves #1882" statement in the original message of this PR, which I believe is sufficient for automatically closing the issue once this PR is closed. For good measure I went ahead and also added a blank commit with the statement as you mentioned. I didn't even know this could be done -- thanks for the info.

**Comment by liangfok** [comment]
@david-german-tri: PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+====================
+
+Introduction
+------------
+Drake is undergoing continuous development and cannot yet offer a stable, documented API or ABI. To help downstream users cope, this document provides brief summaries of the changes. The changes are organized by pull requests that are merged into Drake's master branch.
+
+Descriptions of Pull Requests
+------------------------------
+
+### ([@liangfok][], [#1953][]) - Replacement of valuecheckMatrix() with CompareMatrices()
```
Comment (david-german-tri) [inline CHANGELOG.md]: This is pretty cool, didn't know Sphinx and GitHub were integrated like that.

**Comment by david-german-tri** [comment]
LGTM, ready to merge once CI is green

**Comment by wxmerkt** [comment]
@liangfok Yes, that would have worked as well (my bad, didn't see the (i) icon indicating it would close on the original issue).

**Comment by liangfok** [comment]
+1 for having a team for the open-humanoids collaborators

**Comment by liangfok** [comment]
The CI test results indicate a problem with one of the Matlab unit tests (gcc --> linux_matlab --> systems/test/linear_gaussian_dt).

```
Error using valuecheck (line 78)
Values don't match.  Expected 
1.7597420910651
 but got 
2

Error in linear_gaussian_dt (line 14)
valuecheck(2,var(eval(xtraj,getBreaks(xtraj))),.2);

Error in fevalPackageSafe (line 21)
  feval(f);
```
Since this PR does not involve any code, I'm suspicious about the error. I'm going to close and then reopen this PR to see if the problem is repeatable.

**Comment by liangfok** [comment]
The "required" CI test passed the second time around — not sure why it initially failed. Since this is a relatively safe PR, it passed the required CI test, and @david-german-tri previously approved it, I shall proceed to merge it.

**Pull request merged**